servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -728,6 +739,10 @@ private TypeSpec.Builder addServiceFactory(final State state, final TypeSpec.Bui
                         .addJavadoc(JAVADOC_PARAM + strategyFactory +
                                 " a factory that creates an execution strategy for different {@link $L#id() id}s" +
                                 lineSeparator(), RouteExecutionStrategy)
+                        .addJavadoc(JAVADOC_DEPRECATED + "Use {@link $L#$L()} and set the custom strategy " +
+                                "on {@link $L#$L($T)} instead." + lineSeparator(), Builder, Builder, Builder,
```

Comment:
Consider also changing internals of this constructor as suggested in the javadoc. It will be easier for us to remove deprecated code with less changes later.

Reply:
Actually, this ctor is also deprecated. Nvm. Let's keep as-is.